readthedocs/rtd_tests/tests/test_urls.py

```diff
@@ -5,52 +5,53 @@
 from readthedocs.builds.constants import LATEST
 import readthedocs.core.views
 
+
```

Comment:
Sorry, this got edited in dev & had some whitespace changes. No actual code changes.